web/src/main/java/com/epam/rft/atsy/web/configuration/WebConfiguration.java

```diff
@@ -24,14 +28,31 @@
 import org.springframework.web.servlet.view.InternalResourceViewResolver;
 import org.springframework.web.servlet.view.json.MappingJackson2JsonView;
 
+import java.io.File;
 import java.util.List;
 import java.util.Locale;
 
+import javax.annotation.PostConstruct;
+
 @Configuration
 @ComponentScan("com.epam.rft.atsy.web")
+@PropertySource("classpath:file_storage.properties")
 @Import({ServiceConfiguration.class})
 public class WebConfiguration extends DelegatingWebMvcConfiguration {
 
+  @Value("${upload_location_cv}")
+  private String uploadLocationValue;
+  public static String UPLOAD_LOCATION;
```

Comment:
In my opinion this solution with static field and with FileUploadingUtil is an unnecessary magic. Fortunately there is a simpler solution: it is possible to use @Value annotation in controller classes, so FileUploadingUtil should be eliminated and use @Value("${upload_location_cv}") in FileDownloadController and FileUploadController. There is another static field in FileUploadingUtil: CV_NAME. This field is only used in CandidateCreationController, so my suggestion: move this constant to CandidateCreationController and delete FileUploadingUtil class.